bitnami/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 6.27.0
+version: 6.27.1
```

Comment:
Please bump the minor version as you are adding a new feature.

Reply:
Yea sorry about this, working on the merge but it's taking me a bit longer than normal because a new major version of the chart dropped while I was making these changes (bad luck).

I'm making a few changes to the PR based on the new major chart rev (they renamed `nodeTlsPort` -> `tlsNodePort` which I like a lot, so I've changed all my ports to `xxxNodePort`. I figure now's the time to make that change since it only affects me and isn't in the baseline yet.

Reply:
Resolved In Patch, bumped to `7.1.0`